package.json

```diff
@@ -26,9 +26,7 @@
     "@theforeman/vendor": ">= 6.0.0"
   },
   "dependencies": {
-    "c3": "^0.4.11",
-    "humanize-duration": "^3.20.1",
-    "react-intl": "^2.8.0"
```

Comment:
Looks like this one wasn't used in this repo

Reply:
Looks like it was removed here: https://github.com/theforeman/foreman-tasks/pull/453